test/eos_ssh/test_heredoc.py

```diff
@@ -0,0 +1,190 @@
+import mock
```

Comment:
should be `from unittest import mock`

https://docs.python.org/3/library/unittest.mock.html